src/@next/components/atoms/Select/Select.tsx

```diff
@@ -36,8 +39,11 @@ const optionStyle = (customTheme: any) => ({
 export const Select: React.FC<IProps> = ({
   value,
   onChange,
+  isClearable,
```

Comment:
I'd refrain from using **is**Something naming scheme for component props, it's less arbitary. Think `<Component someProp=val disabled />` vs `<Component someProp=val isDisabled />`

Reply:
**is**Something is the preferred naming scheme for functions that return booleans
```
isProductAvailable = (product) => product.available && product.count > 0
``` 
Let me know how you feel about this, since it's all a convention and can go either way

Reply:
Seems to be reasonable, I'll change it